framework/CodeInterfaces/MELCOR/MELCORdata.py

```diff
@@ -56,6 +62,29 @@ def getTimeBlocks(self):
       timeBlock[info[1]] = self.lines[info[0]+1:endLineCnt]
     return timeBlock
 
+  def returnControlFunctions(self, timeBlock):
+    """
+      CONTROL FUNCTIONS EDIT
+      @ In, timeBlock, dict, {"time":[lines Of Output for that time]}
```

Comment:
The  output description is not present. Please modify the docstring as follows:

```
      """
        CONTROL FUNCTIONS EDIT
        @ In, timeBlock, dict, {"time":[lines Of Output for that time]}
        @ Out, functionValuesForEachTime, dict, ***explanation of the format of the dictionary and variables***
      """
```

Reply:
I believe this is done, please let me know if the @Out description I've added is not what was needed.